### DISCUSSION TITLE
databend  lack some clickhouse functions

### DISCUSSION THREAD

**Discussion by lddlww** [databendlabs/databend#10378] [Q&A]
databend version：
```
 8.0.26-v0.9.16-nightly-3fcebe7(rust-1.68.0-nightly-2023-01-27T16:29:17.399818523Z)
```

lack function:
```
windowfunnel
toUnixTimestamp
```

**Comment by bohutang** [comment]
Databend function name style is not the same as ClickHouse:
windowfunnel : https://databend.rs/doc/sql-functions/aggregate-functions/aggregate-windowfunnel

`toUnixTimestamp`  I would cc @sundy-li  for help :)

**Reply by lddlww** [reply]
thank you，windowfunnel in databend is window_funnel

**Reply by sundy-li** [reply]
we can cast timestamp to int64 (microseconds), if you want to have seconds you can use

```
MySQL [(none)]> select now()::Int64/1e6;
+--------------------------+
| (now()::int64 / 1000000) |
+--------------------------+
|               1678091806 |
+--------------------------+
1 row in set (0.006 sec)

```
